### ISSUE TITLE
Dont know what to do anymore

### ISSUE THREAD

**Issue by RoSo7** [dhleong/ps4-waker#155]
Hey!! I'm havind a weird problem. As soon as I rest my PS4, unless he's downloading something, it disconnects from the network, and so, the program cant work. I dont know what to do anymore, I've searched every forum, every issue, and cant find any solution. Please Help

**Comment by dhleong** [comment]
Please don't open a new issue unless it's actually a different problem
